Movie/MovieTests/ListScreenMocks/MockCacheService.swift

```diff
@@ -0,0 +1,22 @@
+// MockCacheService.swift
+// Copyright © Alexandr Grigorenko. All rights reserved.
+
+import Foundation
+@testable import Movie
+
+// Моковый класс CacheService
+final class MockCacheService: CacheServiceProtocol {
+    // MARK: - Private properties
+
+    private var cachedDataMap: [String: Data] = [:]
+
+    // MARK: - Public methods
+
+    func saveDataToCache(fileURL: String, data: Data, cacheDataType: Movie.CacheDataType) {
```

Comment:
```suggestion
    func saveDataToCache(fileURL: String, data: Data, cacheDataType: CacheDataType) {
```
Это опционально, но как по мне читаемость повышает